lib/eslint/eslint.js

```diff
@@ -389,6 +389,20 @@ function getFixerForFixTypes(fix, fixTypesSet, config) {
 		originalFix(message);
 }
 
+/**
+ * Retrieves flags from the environment variable ESLINT_FLAGS.
+ * @param {string[]} flags The flags defined via the API.
+ * @returns {string[]} The merged flags to use.
+ */
+function mergeEnvironmentFlags(flags) {
+	if (!process.env.ESLINT_FLAGS) {
+		return flags;
+	}
+
+	const envFlags = process.env.ESLINT_FLAGS.split(/\s*,\s*/gu);
```

Comment:
Since we are removing space characters between a flag and a comma I think it makes sense to also remove leading and trailing spaces from the environment variable for consistency.